Makefile.base.mk

```diff
@@ -216,8 +216,15 @@ ifeq ($(HAVE_CAIRO),true)
 
 DGL_FLAGS   += -DHAVE_CAIRO
 
+ifneq ($(WINDOWS),true)
 CAIRO_FLAGS  = $(shell pkg-config --cflags cairo)
 CAIRO_LIBS   = $(shell pkg-config --libs cairo)
+else
+# Always build statically on windows
+CAIRO_FLAGS  = $(shell pkg-config --static --cflags cairo)
+CAIRO_LIBS   = $(shell pkg-config --static --libs cairo)
+CAIRO_LIBS   += -lgdi32
```

Comment:
This is not supposed to be here, gdi32 is already used in `DGL_LIBS`.
If something fails to build, I missed adding that variable..

Reply:
It's because the link step should place the system libs after the cairo libs on the linker line.
Unfortunately, it's `-lgdi32` which gets listed first, and so this fails.

Reply:
If it needs `-lgdi32`, something is wrong then. Is it used by cairo somewhere?

Reply:
GDI is used by cairo when win32 surfaces are built in.
It's kinda a cairo-side issue, because `cairo-win32.pc` does not indicate GDI in the library array; so it's necessary that I mention it in a static link setting.

Unless it is linked after cairo, you will observe a series of `undefined references` from `libcairo.a`.

I believe it will be fixed if system libs are collected in an intermediate variable, eg.
```
DGL_LIBS  += $(CAIRO_LIBS)
DGL_LIBS  += $(DGL_SYSTEM_LIBS)
```

Reply:
I see.. I was afraid that was the issue.
I will think of something

Reply:
Implemented a solution in 2cf5b89